### PR TITLE
Edits to  Postgres Distributed for Kubernetes v1.0.1 #6152

### PR DESCRIPTION
**Pull request by ebgitelman**

### CODE REVIEW

File: product_docs/docs/postgres_distributed_for_kubernetes/1/backup.mdx [hunk 1]
```diff
@@ -123,28 +123,28 @@ spec:
...
+-   **cluster** &mdash; Sets the cluster as owner of the backup.
 
 !!! Warning
-    The `.spec.backup.cron` field is now deprecated. Please use
+    The `.spec.backup.cron` field is deprecated. Use
     `.spec.backup.schedulers` instead.
-    Note that while `.spec.backup.cron` can still be used, it cannot
-    be used simultaneously with `.spec.backup.schedulers`.
+    While you can still use `.spec.backup.cron`, you can't use it
+    with `.spec.backup.schedulers`.
```
Comment (ebgitelman) [inline product_docs/docs/postgres_distributed_for_kubernetes/1/backup.mdx:143]:
```suggestion
    at the same time as `.spec.backup.schedulers`.
```

File: product_docs/docs/postgres_distributed_for_kubernetes/1/ldap.mdx [hunk 2]
```diff
@@ -31,10 +31,10 @@ spec:
...
 
-Here is a example of LDAP configuration using `search+bind` mode in PGDGroup. 
-In this mode, the postgres is first bound to the LDAP using `bindDN` with its password stored
-in the secret `bindPassword`, then postgres tries to perform a search under `baseDN` to find a 
-username that matches the item specified by `searchAttribute`, if a match is found, postgres finally 
+This example shows configuring LDAP using `search+bind` mode in PGDGroup.
+In this mode, the postgres database is first bound to the LDAP using `bindDN` with its password stored
+in the secret `bindPassword`. Then Postgres tries to perform a search under `baseDN` to find a 
+username that matches the item specified by `searchAttribute`. If a match is found, Postgres finally 
 verifies the entry and the password against the LDAP server.
```
Comment (ebgitelman) [inline product_docs/docs/postgres_distributed_for_kubernetes/1/ldap.mdx:38]: Please check the above clarifications of the word "postgres." it looked like the first one might have meant the name of a database and the others referred to the product, but that needs to be checked.

File: product_docs/docs/postgres_distributed_for_kubernetes/1/managed.mdx [hunk 3]
```diff
@@ -1,12 +1,12 @@
 ---
-title: 'Managed Configuration'
+title: 'Managed configuration'
 originalFilePath: 'src/managed.md'
 ---
 
-The PGD operator allows configuring the `managed` section of a PG4K cluster. The `spec.cnp.managed` stanza
-is used for configuring the supported managed roles within the cluster.
+The PGD operator allows configuring the `managed` section of a PGD4K cluster. The `spec.cnp.managed` stanza
```
Comment (ebgitelman) [inline product_docs/docs/postgres_distributed_for_kubernetes/1/managed.mdx:6]: Please check this edit. Was it meant to say PG4K or PGD4K, which is what this doc is about?

File: product_docs/docs/postgres_distributed_for_kubernetes/1/recovery.mdx [hunk 4]
```diff
@@ -221,14 +221,14 @@ spec:
...
 
-For more information, please see [EDB Postgres for Kubernetes recovery from volumeSnapshot objects](/postgres_for_kubernetes/latest/recovery/#recovery-from-volumesnapshot-objects).
+For more information, see [Recovery from volumeSnapshot objects](/postgres_for_kubernetes/latest/recovery/#recovery-from-volumesnapshot-objects) in the EDB Postgres for Kubernetes documentation.
 
 ## PITR from volumeSnapshot
 
-Same as when doing recovery from an object store, you can instruct PostgreSQL to halt the replay of Write-Ahead Logs (WALs)
-at any specific moment during volumeSnapshot recovery.
+You can instruct PostgreSQL to halt the replay of write-ahead logs (WALs)
+at any specific moment during volumeSnapshot recovery. This is the same capability as when recovering from an object store.
```
Comment (ebgitelman) [inline product_docs/docs/postgres_distributed_for_kubernetes/1/recovery.mdx:229]: Please check this edit. The introductory clause was awkward. I think this is what was meant?

**Comment by ebgitelman** [comment]
@gvasquezvargas @josh-heyer Not sure which one of you needs to review my queries.

File: product_docs/docs/postgres_distributed_for_kubernetes/1/ldap.mdx (hunk 2 above)
Comment (gvasquezvargas) [reply product_docs/docs/postgres_distributed_for_kubernetes/1/ldap.mdx:38]: I think your edit goes in the right direction, but looking at [Postgres LDAP docs](https://www.postgresql.org/docs/current/auth-ldap.html), it seems like the "binding" is done per server, not per database, so I would suggest changing "postgres database" to "Postgres database server".
@jsilvela to confirm

File: product_docs/docs/postgres_distributed_for_kubernetes/1/ldap.mdx [hunk 5]
```diff
@@ -1,22 +1,22 @@
...
+    [LDAP authentication feature in the Postgres documentation](https://www.postgresql.org/docs/current/auth-ldap.html).
 
-With LDAP support, only the user authentication is sent to LDAP, so the user must already exist in the postgres database. 
+With LDAP support, only the user authentication is sent to LDAP, so the user must already exist in the postgres database.
 
-Here is an example of LDAP configuration using `simple bind` mode in PGDGroup, 
-postgres simply use `prefix + username + suffix` and password to bind the LDAP 
+This example shows an LDAP configuration using `simple bind` mode in PGDGroup.
+Use `prefix + username + suffix` and password to bind the LDAP 
 server to achieve the authentication.
```
[the remark this answers is not in the document]
Comment (gvasquezvargas) [reply product_docs/docs/postgres_distributed_for_kubernetes/1/ldap.mdx:20]: I believe here Postgres refers to the server.

I also suspect it is not the user who uses `prefix + username + suffix` to bind the LDAP server, but it is rather the Postgres server that uses `prefix + username + suffix` to bind the LDAP server after the user specifies that they want to use the bind mode. 

Let's loop in @jsilvela :).

**Review by gvasquezvargas — COMMENTED** [review]
Dropped a few comments but tagged one of the engineers on the PGD4K team to confirm.

File: product_docs/docs/postgres_distributed_for_kubernetes/1/recovery.mdx (hunk 4 above)
Comment (jsilvela) [reply product_docs/docs/postgres_distributed_for_kubernetes/1/recovery.mdx:229]: agree with the edit.

File: product_docs/docs/postgres_distributed_for_kubernetes/1/ldap.mdx (hunk 5 above)
[the remark this answers is not in the document]
Comment (jsilvela) [reply product_docs/docs/postgres_distributed_for_kubernetes/1/ldap.mdx:20]: I agree with Gaby's comment that `prefix + username + suffix` to bind ... is done by the server.
The errant `postgres`, is more a reference to the capabilities of the Postgres database, i.e. we're just passing things along through the layers of PG4K-PGD for Postgres to do the work.

File: product_docs/docs/postgres_distributed_for_kubernetes/1/ldap.mdx (hunk 2 above)
Comment (jsilvela) [reply product_docs/docs/postgres_distributed_for_kubernetes/1/ldap.mdx:38]: Agree with Gaby's comment, but also, "server", like "database", is already pretty overloaded.
In the context the Kubernetes products we tend to talk more about instances and pods.
Like the other question about the "errant Postgres", this is meant to call out that the actual work is performed by Postgres.
I know I just used another errant Postgres, I know.
In other parts of the CNPG / PG4K documentation we distinguish operator from operand.
Might it be useful here? I.e. the bind is performed at the operand level.

File: product_docs/docs/postgres_distributed_for_kubernetes/1/managed.mdx (hunk 3 above)
Comment (jsilvela) [reply product_docs/docs/postgres_distributed_for_kubernetes/1/managed.mdx:6]: It was meant for PG4K, without the D.
The confusing bit here: PGD4K ... let me call it PG4K-PGD for better contrast ... leverages the PG4K operator under the covers. The thing the PG4K operator manages is "clusters". In PG4K-PGD we tend to manage PGD Groups.
Hope that helps.

Comment (jsilvela) [reply product_docs/docs/postgres_distributed_for_kubernetes/1/managed.mdx:6]: The `spec.cnp.managed` stanza allows us to pass some PG4K parameters along to the PG4K "level".

File: product_docs/docs/postgres_distributed_for_kubernetes/1/pg4k-pgd.v1beta1.mdx [hunk 6]
```diff
@@ -1,11 +1,11 @@
...
-title: 'API Reference'
+title: 'API reference'
 originalFilePath: 'src/pg4k-pgd.v1beta1.md'
 ---
 
-<p>Package v1beta1 contains API Schema definitions for the pgd v1beta1 API group</p>
+<p>Package v1beta1 contains API schema definitions for the pgd v1beta1 API group.</p>
 
-## Resource Types
+## Resource types
```
[the remark this answers is not in the document]
Comment (jsilvela) [reply product_docs/docs/postgres_distributed_for_kubernetes/1/pg4k-pgd.v1beta1.mdx:9]: Given that this is a documentation repo and in the development repo we don't manage `.mdx` files, not 100% what to tell you here. The file *is* autogenerated in the development repo, though, from code annotations and comments.

File: product_docs/docs/postgres_distributed_for_kubernetes/1/ldap.mdx (hunk 2 above)
Comment (gvasquezvargas) [reply product_docs/docs/postgres_distributed_for_kubernetes/1/ldap.mdx:38]: @ebgitelman  I think in that case we have two options: 

- We can talk about Postgres as the technology performing these kind of operations. If we want to remain general, we could say that the "Postgres instance tries to perform a search under ...". 
- We can be specific and say that the operand is the entity performing the binding and link to https://www.enterprisedb.com/docs/postgres_distributed_for_kubernetes/latest/identify_images/identify_image_name/#operand-image-name. Although this is technically more accurate, I am not sure we would be introducing more terms here that are potentially confusing without bringing much value. 

WDYT?

File: product_docs/docs/postgres_distributed_for_kubernetes/1/pg4k-pgd.v1beta1.mdx (hunk 6 above)
[the remark this answers is not in the document]
Comment (gvasquezvargas) [reply product_docs/docs/postgres_distributed_for_kubernetes/1/pg4k-pgd.v1beta1.mdx:9]: @josh-heyer  if this page is generated in the development repo, I believe Betsy's edits will disappear eventually when there is another sync, right? Which means, it is probably not worth it editing?

File: product_docs/docs/postgres_distributed_for_kubernetes/1/ldap.mdx [hunk 7]
```diff
@@ -1,22 +1,22 @@
...
+    Before you proceed, familiarize yourself with the
+    [LDAP authentication feature in the Postgres documentation](https://www.postgresql.org/docs/current/auth-ldap.html).
 
-With LDAP support, only the user authentication is sent to LDAP, so the user must already exist in the postgres database. 
+With LDAP support, only the user authentication is sent to LDAP, so the user must already exist in the postgres database.
 
-Here is an example of LDAP configuration using `simple bind` mode in PGDGroup, 
-postgres simply use `prefix + username + suffix` and password to bind the LDAP 
+This example shows an LDAP configuration using `simple bind` mode in PGDGroup.
+Use `prefix + username + suffix` and password to bind the LDAP
```
Comment (ebgitelman) [inline product_docs/docs/postgres_distributed_for_kubernetes/1/ldap.mdx:19]:
```suggestion
The Postgres server uses `prefix + username + suffix` and password to bind the LDAP 
```

File: product_docs/docs/postgres_distributed_for_kubernetes/1/ldap.mdx (hunk 5 above)
[the remark this answers is not in the document]
Comment (ebgitelman) [reply product_docs/docs/postgres_distributed_for_kubernetes/1/ldap.mdx:20]: Is this the correct edit then?

File: product_docs/docs/postgres_distributed_for_kubernetes/1/ldap.mdx (hunk 2 above)
Comment (ebgitelman) [reply product_docs/docs/postgres_distributed_for_kubernetes/1/ldap.mdx:38]: Let's go with the first option.

File: product_docs/docs/postgres_distributed_for_kubernetes/1/ldap.mdx [hunk 8]
```diff
@@ -31,10 +31,10 @@ spec:
           suffix: ",dc=example,dc=org"
 ```
 
-Here is a example of LDAP configuration using `search+bind` mode in PGDGroup. 
-In this mode, the postgres is first bound to the LDAP using `bindDN` with its password stored
-in the secret `bindPassword`, then postgres tries to perform a search under `baseDN` to find a 
-username that matches the item specified by `searchAttribute`, if a match is found, postgres finally 
+This example shows configuring LDAP using `search+bind` mode in PGDGroup.
+In this mode, the postgres database is first bound to the LDAP using `bindDN` with its password stored
```
Comment (ebgitelman) [inline product_docs/docs/postgres_distributed_for_kubernetes/1/ldap.mdx:35]:
```suggestion
In this mode, the Postgres instance is first bound to the LDAP using `bindDN` with its password stored
```

File: product_docs/docs/postgres_distributed_for_kubernetes/1/ldap.mdx (hunk 2 above)
Comment (ebgitelman) [reply product_docs/docs/postgres_distributed_for_kubernetes/1/ldap.mdx:38]: Please check edit and if there's something else you want, please let me know the correct wording.

File: product_docs/docs/postgres_distributed_for_kubernetes/1/managed.mdx (hunk 3 above)
Comment (ebgitelman) [reply product_docs/docs/postgres_distributed_for_kubernetes/1/managed.mdx:6]: Let me know if any other clarification is in order here for the doc.

File: product_docs/docs/postgres_distributed_for_kubernetes/1/pg4k-pgd.v1beta1.mdx (hunk 6 above)
[the remark this answers is not in the document]
Comment (ebgitelman) [reply product_docs/docs/postgres_distributed_for_kubernetes/1/pg4k-pgd.v1beta1.mdx:9]: I edited only this little bit of capitalization, so if it's lost, that's okay.

**Review by gvasquezvargas — APPROVED** [review]
LGTM

**Pull request merged**